src/locales/en.json

```diff
@@ -117,31 +117,31 @@
     "risk_modifier_multiple": "{{multiplier}}x the risk",
     "firsttime": {
       "microcovid_header": "What is a microCOVID?",
-      "microcovid_explanation": "<p>The calculator introduces a new concept, the microCOVID. One microCOVID is a one-in-a-million chance of getting COVID.</p> <p>An activity that’s 20,000 microCOVIDs is very unsafe, as you have a 2% risk of getting COVID <em>every time you do it</em>. An activity that’s 20 microCOVIDs is relatively safe, as you could do it every week for a year and still have only about a 0.1% chance of getting COVID.</p>",
+      "microcovid_explanation": "<p>The calculator introduces a new concept, the microCOVID. One mC is a one-in-a-million chance of getting COVID-19.</p> <p>An activity of 20,000 mCs is very unsafe, as it comparatively means a 2% risk of getting COVID-19 <em>every time you do it</em>. 20 mCs for one activity is relatively safe, as you could do it every week for a year and still have only about a 0.1% chance of getting COVID-19.</p>",
       "budget_header": "How can I manage my risk using a risk budget?",
-      "budget_explanation": "We decided an annual risk budget of a 1% chance of catching COVID per year was reasonable for most healthy people who are not in contact with vulnerable groups. To learn how we arrived at that number, <1>click here</1>. If you want to follow this 1%-risk-per-year budget, you’ll want your weekly budget of activities to sum to less than about 200 microCOVIDs each week. We’ve created a handy <3>spreadsheet</3> that lets you keep track of your activities over time."
+      "budget_explanation": "An annual risk budget of a 1% chance of catching COVID-19 per year is <1>our assumption</1> for what most healthy people not in contact with vulnerable groups consider reasonable. To follow this 1%-risk-per-year budget, your weekly budget should be less than 200 mCs each week. The handy <3>spreadsheet</3> lets you keep track of your activities over time."
```

Comment:
If what you say is rational, the reader will attribute enough credence to it.
What the original does is making a decision for what is rational for normal people, rather than allude to the rationality of normal people. That is how you are liable when you shouldn't be. "Arriving at the number" doesn't change matters.